### PR TITLE
Add environment variable documentation

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/integrator/docker.rst [hunk 1]
```diff
@@ -76,3 +76,56 @@ You can also remove the unused named images, that should be done manually:
 
 * List the images with ``docker images``.
 * Remove the unwanted images with ``docker rmi <image>:<tag>``.
+
+
+Environment variables
+---------------------
+
+The GeoMapFish containers can be curtomise with some environment variable:
```
Comment (jwkaltz) [inline doc/integrator/docker.rst]: ... can be customised with some environment variables:

File: doc/integrator/docker.rst [hunk 2]
```diff
@@ -76,3 +76,56 @@ You can also remove the unused named images, that should be done manually:
...
+
+Config::
+
+ * ``VISIBLE_WEB_HOST``: The web host visible by the browser e.-g.: 'example.com'.
+ * ``VISIBLE_ENTRY_POINT``: The web path visible by the browser e.-g.: '/main/', default is '/'.
+ * ``PGSCHEMA``: The postgres schema, used by mapserver.
+ * ``GEOPORTAL_INTERNAL_URL``: Use by the print in non mutualised mode.
```
Comment (jwkaltz) [inline doc/integrator/docker.rst]: Use -> Used

File: doc/integrator/docker.rst [hunk 3]
```diff
@@ -76,3 +76,56 @@ You can also remove the unused named images, that should be done manually:
...
+ * ``TILECLOUDCHAIN_INTERNAL_URL``: Use by the print in non mutualised mode.
```
Comment (jwkaltz) [inline doc/integrator/docker.rst]: copy-paste error

File: doc/integrator/docker.rst [hunk 4]
```diff
@@ -76,3 +76,56 @@ You can also remove the unused named images, that should be done manually:
...
+ * ``ST_JOIN``: Can be ``ST_Collect`` (default) or ``ST_Union``, ``ST_Collect`` is better for performance but he
```
Comment (jwkaltz) [inline doc/integrator/docker.rst]: "he don't support" -> "does not support"

File: doc/integrator/docker.rst [hunk 5]
```diff
@@ -76,3 +76,56 @@ You can also remove the unused named images, that should be done manually:
...
+ * ``GUNICORN_LOG_LEVEL``: Log level for Gunicorn, default is ``INFO``.
+ * ``SQL_LOG_LEVEL``: Log level for the Sqlalchemy engine``, default is ``WARN``.
+ * ``DOGPILECACHE_LOG_LEVEL``: Log level for Dogpile cache, default is ``INFO``.
+ * ``OTHER_LOG_LEVEL``: Log level for other modules, default is ``WARN``.
+
+QGIS server::
+
+ * ``GEOMAPFISH_CONFIG``: The GeoMapFish config file, default is ``/etc/qgisserver/geomapfish.yaml``.
+ * ``GEOMAPFISH_OGCSERVER``: The OGC server name in single QGIS project mode.
+ * ``GEOMAPFISH_ACCESSCONTROL_CONFIG``: The access control config file forn multy QGIS project mode.
```
Comment (jwkaltz) [inline doc/integrator/docker.rst]: "forn multy" -> "from multi" ?

**Comment by sbrunner** [comment]
UP :-)

**Pull request merged**